modules/sharedaddy/sharing.css

```diff
@@ -564,6 +564,38 @@ body .sd-social-icon .sd-content li.share-custom a span {
 }
 
 
+/**
+ * Screen Reader Text for "Icon Only" option
+ */
+
+.sharing-screen-reader-text {
+	clip: rect(1px, 1px, 1px, 1px);
+	position: absolute !important;
+	height: 1px;
+	width: 1px;
+	overflow: hidden;
```

Comment:
I don't think it's needed. I based this on the .screen-reader-text css used in [_s](https://github.com/Automattic/_s/blob/master/style.css#L631-L659) which has been widely tested and seems to work well (unless you've found an issue). As it happens there was a recent post on [Make WordPress Accessible](https://make.wordpress.org/accessibility/2015/02/09/hiding-text-for-screen-readers-with-wordpress-core/) that also covers the subject well.